netcode.c

```diff
@@ -5346,6 +5410,7 @@ static void test_address()
 #define TEST_SERVER_PORT            40000
 #define TEST_CONNECT_TOKEN_EXPIRY   30
 #define TEST_TIMEOUT_SECONDS        15
+#define TEST_MATCH_EXPIRE	1.0
```

Comment:
nit spacing. You might to use vim to check the spacing and modify directly on vim for this, sometime, changes using IDE doesn't match up the actual result for some reasons.